frontend/src/pages/Auth/SignIn.tsx

```diff
@@ -0,0 +1,141 @@
+import { Button } from '@components/Button'
+import {
+	Box,
+	Form,
+	Heading,
+	IconSolidSparkles,
+	Stack,
+	Text,
+	useFormState,
+} from '@highlight-run/ui'
+import SvgHighlightLogoOnLight from '@icons/HighlightLogoOnLight'
+import { AuthBody, AuthError, AuthFooter, AuthHeader } from '@pages/Auth/Layout'
+import { auth } from '@util/auth'
+import firebase from 'firebase/app'
+import React, { useCallback } from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+
+type Props = {
+	setResolver: React.Dispatch<
+		React.SetStateAction<firebase.auth.MultiFactorResolver | undefined>
+	>
+}
+
+export const SignIn: React.FC<Props> = ({ setResolver }) => {
+	const navigate = useNavigate()
+	const [loading, setLoading] = React.useState(false)
+	const [error, setError] = React.useState('')
+	const formState = useFormState({
+		defaultValues: {
+			email: '',
+			password: '',
+		},
+	})
+
+	const handleAuthError = useCallback(
+		(error: firebase.auth.MultiFactorError) => {
+			let errorMessage = error.message
+
+			if (error.code == 'auth/multi-factor-auth-required') {
+				setResolver(error.resolver)
+				navigate('/multi_factor')
+				return
+			}
+
+			if (error.code === 'auth/popup-closed-by-user') {
+				errorMessage =
+					'Pop-up closed without successfully authenticating. Please try again.'
+			}
+
+			setError(errorMessage)
+		},
+		[navigate, setResolver],
+	)
+
+	return (
+		<Form
+			state={formState}
+			resetOnSubmit={false}
+			onSubmit={() => {
+				setLoading(true)
+
+				auth.signInWithEmailAndPassword(
+					formState.values.email,
+					formState.values.password,
+				)
+					.then(() => {})
+					.catch(handleAuthError)
+			}}
+		>
+			<AuthHeader>
+				<Box mb="4">
+					<Stack direction="column" gap="16" align="center">
+						<SvgHighlightLogoOnLight height="48" width="48" />
+						<Heading level="h4">Welcome back.</Heading>
+						<Text>
+							New here?{' '}
+							<Link to="/sign_up">Create an account</Link>.
+						</Text>
+					</Stack>
+				</Box>
+			</AuthHeader>
+			<AuthBody>
+				<Stack gap="12">
+					<Form.Input
+						name={formState.names.email}
+						label="Email"
+						type="email"
+						autoFocus
+					/>
+					<Form.Input
```

Comment:
Should we add `autocomplete="new-password"` here and `autocomplete="current-password"` elsewhere? Chrome issues a warning:
![Screenshot 2023-02-17 at 12 07 46 PM](https://user-images.githubusercontent.com/17913919/219784088-0c755953-995d-46c8-be0d-085519534b6c.jpg)

Reply:
Set this on all the email and password fields here. Good call!